build.psm1

```diff
@@ -447,20 +447,24 @@ function Start-PSBuild {
     }
 
     function Stop-DevPowerShell {
+        if (-not (Test-Path $script:Options.Output)) {
+            return
+        }
+        $pwshDev = Resolve-Path $script:Options.Output
         Get-Process pwsh* |
             Where-Object {
                 $_.Modules |
                 Where-Object {
-                    $_.FileName -eq (Resolve-Path $script:Options.Output).Path
+                    $_.FileName -eq $pwshDev.Path
```

Comment:
Pascal case here too.

Reply:
this is the correct casing for a variable.

Reply:
Sorry, I am confused. Is `PwshDev` or `pwshDev` the correct casing?

Reply:
@kwkam  `PwshDev`

Reply:
In C# typically `camelCase` is considered correct. Largely in PS `$PascalCase` is considered better for variables, though it can come down to style in some cases.